app/components/LiquidityPoolTable.tsx

```diff
@@ -0,0 +1,125 @@
+import Table from 'react-bootstrap/Table'
+import { FormattedMessage } from 'react-intl'
+import type { Horizon } from 'stellar-sdk'
+
+import { type LiquidityPoolProps } from './operations/LiquidityPool'
+import { liquidityPoolAsset } from '~/data/liquidity_pool_asset'
+import { formatAmountToHumanReadable, getAssetCode } from '~/lib/utilities'
+
+import styles from './LiquidityPoolTable.module.css'
+
+interface ParentProps {
+  compact: boolean
+  // horizonURL: string
```

Comment:
[note]

I don't leave commented-out lines normally (I delete them), but I left some this time.
Because it's still sort of half-finished and they might be necessary when we'll work on extending the view.
But it's fine by me to remove them.

Reply:
sure, if it's useful later leave in. that's good.